pkg/cli/import_test.go

```diff
@@ -41,24 +40,34 @@ func runImportCLICommand(
 
 	var out string
 	var err error
-	var wg sync.WaitGroup
-	wg.Add(1)
 
+	errCh := make(chan error)
+	defer close(errCh)
 	go func() {
 		out, err = c.RunWithCapture(cliCmd)
-		require.NoError(t, err)
-		wg.Done()
+		errCh <- err
```

Comment:
This will leak a goroutine if the code below stops (e.g. at require.NoError) before it reads.

You can avert that:
- by adding a buffer of 1 in the channel in the `make` above, OR
- by closing the channel here after writing to it and deferring a read from the channel in the function above before creating the goroutine

Reply:
oh interesting, in the buffer of 1 case, I think I also need to remove the `defer close(errCh)` too? Otherwise, the `require.NoError` would cause the test to return, closing the channel, and then the goroutine would panic on write to a closed channel.

Reply:
you only `close` inside the goroutine, not the outer function. (With a defer to tolerate panics inside the goroutine)